common/templates/license.html

```diff
@@ -0,0 +1,55 @@
+<%page args="license, button=False, button_size='88x31'"/>
+## keep this synchronized with the contents of cms/templates/js/license-selector.underscore
+<%!
+from django.utils.translation import ugettext as _
+
+def parse_license(lic):
```

Comment:
I don't like having functions defined in Mako templates because they don't get included in code coverage, and it is hard to unit test them. Can these be moved out into a helper class?